src/test/java/com/jayway/asyncservlet/ApplicationTests.java

```diff
@@ -30,6 +39,22 @@ public void shouldPass_springBootRepo() throws Exception {
         log.info(result.toString());
 
         assertTrue(op.getStatusCode().is2xxSuccessful());
-
+    }
+    
+    @Import(Application.class)
```

Comment:
Not sure about this - it looks like mixing config with the test class. I usually have the config in src/test/java/.../config/Config.java. Never seen it done like this.